autogpt_platform/backend/backend/integrations/webhooks/__init__.py

```diff
@@ -1,5 +1,8 @@
 from typing import TYPE_CHECKING
 
+from backend.integrations.webhooks.simple_webhook_manager import CompassWebhookManager, SimpleWebhooksManager
+from pyparsing import C
```

Comment:
auto imports strike again

Reply:
what's next, the rest of the alphabet? :+)